Library/Services/INfieldDeliveryRepositorySurveysService.cs

```diff
@@ -30,5 +30,26 @@ public interface INfieldDeliveryRepositorySurveysService
         /// <param name="repositoryId">The repository id.</param>
         /// <returns>The repository surveys.</returns>         
         Task<IQueryable<RepositorySurveyModel>> QueryAsync(long repositoryId);
+
+        /// <summary>
+        /// Adds the selected surveys to the repository.
+        /// </summary>
+        /// <param name="repositoryId">The repository id.</param>
+        /// <param name="nfieldSurveyIds">The surveys to add</param>
+        Task PostAsync(long repositoryId, string[] nfieldSurveyIds);
+
+        /// <summary>
+        /// Reinitiates a paused Survey in the selected Repository.
+        /// </summary>
+        /// <param name="repositoryId">The repository id.</param>
+        /// <param name="surveyId">The id of the survey to be reinitiated</param>
+        Task PutReinitiateAsync(long repositoryId, string surveyId);
```

Comment:
Shouldn't this be called `ReinitiateAsync`? The `Put` seems to be an implementation detail on he API that does not have to leak here

Reply:
It was added as it is like that also in the Public Api but it can be omitted